temba/api/v2/tests.py

```diff
@@ -266,34 +267,63 @@ def api_request(endpoint, token):
                 HTTP_AUTHORIZATION="Token %s" % token,
             )
 
+        def api_request_basic_auth(endpoint, username, token):
+            encoded = "Basic {}".format(base64.encodebytes("{}:{}".format(username, token).encode()).decode()
+                                        .replace('\n',''))
```

Comment:
Maybe break out some variables here so that it's a bit easier to grok?

Reply:
Changes made. Please check them out. Cheers